### PR TITLE
Added Slovenian translations

### PR DESCRIPTION
**Pull request by Linux-Alex**
## Description
Added Slovenian translations to the project for improved multilingual support.

## Checklist
- [x] I have read and followed the [Contributing Guidelines](../CONTRIBUTING.md).
- [x] I have added necessary tests.
- [x] The project builds and runs without issues.

### CODE REVIEW

**Comment by itz-rj-here** [comment]
@prem-k-r what you think, We should first add the language optimization pr first then look into this?

**Comment by prem-k-r** [comment]
Yeah, coz we have to delete this anyway if merged before

File: languages.js [hunk 1]
```diff
@@ -2004,7 +2004,144 @@ const translations = {
         "failedbackup": "Yedəkləmə uğursuz oldu: ",
         "restorecompleted": "Bərpa uğurla tamamlandı!",
         "restorefailed": "Bərpa uğursuz oldu: ",
-    }
+    },
+
+    // Slovenian (Slovenščina)
```
Comment (prem-k-r) [inline languages.js:2009]: please make a file `sl.js` here and place this there
https://github.com/XengShi/materialYouNewTab/tree/main/locales

File: index.html [hunk 2]
```diff
@@ -1233,6 +1233,7 @@ <h1>Material You NewTab</h1>
                             <option value="tr">Türkçe (Turkish)</option>
                             <option value="uz">O'zbek (Uzbek)</option>
                             <option value="vi">Tiếng Việt (Vietnamese)</option>
+                            <option value="sl">Slovenščina (Slovenian)</option>
```
Comment (prem-k-r) [inline index.html:1236]: Please place this between Russian and Spanish

File: languages.js [hunk 3]
```diff
@@ -2004,7 +2004,144 @@ const translations = {
...
+        "rangColor": "Izberi barvo",    // Keep this short
+
+        // Dialog boxes (alerts)
+        "confirmWallpaper": "Ali želite nastaviti novo sliko kot ozadje danes?",
+        "confirmRestore": "Ali ste prepričani, da želite ponastaviti nastavitve? To dejanje ni mogoče razvejaviti.",
+        "Nobackgroundset": "Trenutno ni nastavljena nobena slika za ozadje.",
+        "clearbackgroundimage": "Ali ste prepričani, da želite odstraniti sliko ozadja?",
+        "ProxyDisclaimer": "Vse funkcije proxyja so privzeto izklopljene.\n\nČe omogočite predloge za iskanje in proxy za obvoz CORS, je močno priporočljivo, da gostite svoj proxy za boljšo zasebnost.\n\nPrivzeto bo proxy nastavljen na https://mynt-proxy.rhythmcorehq.com, kar pomeni, da bodo vsi vaši podatki šli skozi to storitev, kar lahko predstavlja tveganje za zasebnost.",
+        "endlink": "Na koncu povezave en sme biti /",
+        "onlylinks": "Dovoljene so samo spletne URL povezave (ki se začnejo s http:// ali https://).",
```
Comment (prem-k-r) [inline languages.js:2138]: endlink and onlylinks have been deleted recently

**Review by prem-k-r — CHANGES_REQUESTED** [review]
Please search for `fr:` in `script.js`, you will find two results, and add date localisation for both clocks if needed

Also you can add the language and your name in Readme.md
https://github.com/XengShi/materialYouNewTab#-currently-supported-languages-18

**Comment by Linux-Alex** [comment]
Thanks for all the tips. Because of merging conflicts, I will be making a new updated branch and reimplement my language translations

**Pull request closed without merging**